app/models/ems_refresh/save_inventory_cloud.rb

```diff
@@ -171,7 +171,6 @@ def save_key_pairs_inventory(ems, hashes, target = nil)
   def save_cloud_volumes_inventory(ems, hashes, target = nil)
     target = ems if target.nil?
 
-    return unless ems.cloud_volumes
```

Comment:
@hsong-rh Isn't some sort of check still needed here? Otherwise, OpenStack refresh will try to save volume inventory. Can we just rely on the fact that there shouldn't be any `:cloud_volumes`, etc in the hashes?